extensions/linux-source-package.sh

```
@@ -0,0 +1,60 @@
+
+		#( set -o posix ; set )
+		echo ${kernel_version_family}
+
+		display_alert "Packaging kernel source..." "${EXTENSION}" "info"
+		declare kernel_work_dir="${SRC}/cache/sources/${LINUXSOURCEDIR}"
+		declare CHOSEN_KSRC=linux-source-${BRANCH}-${LINUXFAMILY}
+
+		ts=$(date +%s)
+		local sources_pkg_dir tmp_src_dir
+		tmp_src_dir=$(mktemp -d)
+		trap "ret=\$?; rm -rf \"${tmp_src_dir}\" ; exit \$ret" 0 1 2 3 15
```

Shellcheck (0.10.0)

[warning] ret is referenced but not assigned.

(SC2154)